test/Feature/InsertSpec.hs

```diff
@@ -537,3 +537,13 @@ spec actualPgVersion = do
             , matchHeaders = [ "Location" <:> "/compound_pk_view?k1=eq.1&k2=eq.test"
                              , "Content-Range" <:> "*/*" ]
             }
+
+      it "should not throw and return location header when a PK is null" $
+        request methodPost "/test_null_pk_competitors_sponsors" [("Prefer", "return=headers-only")]
+            [json|{"id":1}|]
+          `shouldRespondWith`
+            ""
+            { matchStatus  = 201
+            , matchHeaders = [ "Location" <:> "/test_null_pk_competitors_sponsors?id=eq.1&sponsor_id=is.null"
```

Comment:
Just to clarify, `test_null_pk_competitors_sponsors` doesn't have`sponsor_id` as a pk column:

```sql
CREATE TABLE competitors (
    id integer primary key,
    sponsor_id integer references sponsors(id),
    full_name varchar(40) not null
);

CREATE VIEW test_null_pk_competitors_sponsors  AS
SELECT c.id, s.id as sponsor_id
FROM competitors c
    LEFT JOIN sponsors s on c.sponsor_id = s.id;
```

So the above would be a bug.